devel/cctools/Portfile

```diff
@@ -88,45 +86,79 @@ if {![some_llvm_variant_set]} {
     #     default_variants +llvm33
     # }
 
-    if {![some_llvm_variant_set] && ${os.major} >= 10} {
+    if {[vercmp $xcodeversion 10.0] >= 0} {
+        default_variants +xcode
+    } elseif {![some_llvm_variant_set] && ${os.major} >= 10} {
         default_variants +llvm50
     }
+
+}
+
+variant xcode description "Use Xcode supplied toolkit" { }
+if {[variant_isset xcode]} {
+   patch { }
+   extract { }
+   fetch { }
+   build { }
+   pre-destroot { }
+   post-destroot { }
+   post-extract { }
+   checksum { }
+   destroot {
```

Comment:
You should basically never override the fetch, checksum or extract phases to disable them; instead, clear `distfiles`.

Reply:
OK. Could you give me a pointer though on the syntax to do this. Can't seem to find an example.

Reply:
No worries, eventually guessed it.